charters/wot-wg-2023-draft.html

```diff
@@ -491,6 +491,81 @@ <h3 id="w3c-coordination">W3C Groups</h3>
 
         <section>
           <h3 id="external-coordination">External Organizations</h3>
+          <p>
+            To succeed in establishing inter-platform standards,
+            W3C needs to coordinate with IoT alliances and standards development organizations.
+          </p>
+          <p>
+            List of new liaisons we want to form
+          </p>
+          <dl>
+            <dt> <a href="https://opcfoundation.org/">OPC Foundation</a>
+              <dd>
+                For coordination on the development of the OPC-UA Binding for the W3C Web of Things via the joint working group.
+                A Memorandum of Understanding (MoU) / Multi-Organization Cooperation Agreement (MOCA) will be signed.
+              </dd>
+            </dt>
+            <dt> <a href="https://industrialdigitaltwin.org/en/">Industrial Digital Twin Association</a>
+              <dd>
+                For coordination on the use of TDs and Binding Templates in the Asset Interface Description (AID) Submodel for the Asset Administration (AAS) Specification.
+                Additionally, the AAS Submodels can be used for Digital Nameplate and Digital Product Passport of products that the European Commission is demanding each product to supply.
+                Also see <a href="https://github.com/w3c/wot/issues/978#issuecomment-1359830278">the related GitHub comment</a>.
+              </dd>
+            </dt>
+            <dt> <a href="https://www.ashrae.org/">ASHRAE</a>
+              <dd>
+                For coordination on the development of the BACnet (ASHRAE 135-2020) Binding for the W3C Web of Things .
+              </dd>
+            </dt>
+            <dt><a href="https://github.com/Azure/opendigitaltwins-dtdl">Azure/Microsoft/OpenDigitalTwins DTDL</a></dt>
```

Comment:
How do we deal with specs like this that are from one company?  A kind of "de facto" status...

Reply:
Maybe list under "other collaborations"